source/wp-content/themes/wporg-parent-2021/patterns/404-page-subtitle.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Title: 404 Page Subtitle
+ * Slug: wporg-parent-2021/404-page-subtitle
+ * Categories: wporg
+ */
+
+?>
+
+<!-- wp:paragraph -->
+<p>
+	<?php esc_html_e( 'Go to', 'wporg' ); ?>
+	<a href="https://wordpress.org"><?php esc_html_e( 'the homepage', 'wporg' ); ?></a>,
+	<?php esc_html_e( 'or try searching WordPress.org sites using the field below.', 'wporg' ); ?>
```

Comment:
Since this is one sentence, it should be all in the same translation call. HTML can be used & sanitized with `wp_kses_post`. Also, maybe we can use `get_site_url` instead of hardcoding wp.org, which should return the user to the homepage of the site they're currently on.

```suggestion
	<?php
	printf(
		/* translators: %s is the site URL. */
		wp_kses_post( __( 'Go to <a href="%s">the homepage</a> or try searching WordPress.org sites using the field below.', 'wporg' ) ),
		esc_url( get_site_url( '/' ) )
	);
	?>
```